docs/_data/components/schemas/array-max-forecast-periods.yaml

```diff
@@ -30,9 +30,13 @@ forecast-proposal-for-segment:
     periods:
       <<: *max
       items:
-        $ref: ./forecast-rating-period.yaml
+        $ref: 'forecast-limit-period.yaml#/forecast-period-proposal'
 
-missing-forecast-rating-for-segment:
+  required:
+    - segment-id
```

Comment:
I _think_ I'm inclined to agree, but there's an interesting challenge here with interop- whether you call it segments and facilities or not, it seems likely to me that you almost _have_ to consider that the thing you submit proposals against and the thing that you get snapshots against are probably different.

Reply:
Quoting a definition I proffered from a [different discussion](https://github.com/trolie/spec/issues/45#issue-2115655034), "A _Segment_ is a Power System Object representing all of the series-connected (not necessarily contiguous) equipment in a Transmission Facility that _is the responsibility of a particular Ratings Provider_ under the implicit topology." (emphasis added)

Interfaces are another Power System Object associated with a particular Ratings Provider. Both kinds of Power System Objects are related to the Ratings Provider via a Ratings Obligation.

A third Power System Object is the Transmission Facility. Segments are related to the Transmission Facility via a Ratings Obligation.

If I look at a Ratings Obligation profile, I should see one of these hierarchies or similar:
```
Ratings Provider
|__Segment (multiple)
|   |__Transmission Facility (one)
|__ Interface (multiple)
```
Alternatively,
```
Transmission Facility (multiple)
|__ Segments (multiple)
|   |__Ratings Provider (one)
|__ Interfaces (multiple)
    |__ Ratings Provider (one)
```
*However*, this could be represented with fewer concepts:
```
Ratings Provider
|__Ratings Obligation (many)

Power System Object
|__Ratings Obligation (many)
   |__Power System Object (one)
```
All Transmission Facilities are related to one or more Ratings Obligations; granted that Ratings Obligation is for a Segment, but topology and the network model in general are not necessarily part of the TROLIE core domain. A vendor extension could add a `object-type: Segment`, but we don't lose interpretability by paring things down to just Ratings Obligations, imo.

A Monitoring Set can then be defined in terms of its constituent Power System Objects:
```
Monitoring Set
|__Power System Object (many)
```

Reply:
There's an inherent asymmetry between proposal and snapshot, i.e., segments-in, facilities-out. The provenance in a detailed snapshot includes the segments-in, but I think most snapshot consumers will only be interested in the facility (or interface) limits. They'll have an mrid or other identifier they use to reference those limits.

Reply:
The CIM parlance is Power System Resource, not Power System Object, so I'd like to revise all of the above to `object`->`resource`, so `resource-id` where I have `object-id`, for example.